src/main/java/de/metas/ui/web/procurement/process/PMM_Purchase_Candidate_CreatePurchaseOrder.java

```diff
@@ -36,27 +32,14 @@
  * Mass enqueue {@link I_PMM_PurchaseCandidate} records to be processed and purchase orders to be generated.
  *
  * @author metas-dev <dev@metasfresh.com>
- *
  */
 @Profile(Profiles.PROFILE_Webui)
 public class PMM_Purchase_Candidate_CreatePurchaseOrder
 		extends ViewBasedProcessTemplate
-		implements IProcessPrecondition
+
 {
 	private int recordsEnqueued;
 
-	@Override
```

Comment:
Removing this check will allow the user to run the process even if no records are selected.
But the process when it runs will process only the currently selected records.

because of 
```java
.filter(getProcessInfo().getQueryFilterOrElseFalse())
```

which we have below.